app/js/controllers/foyer/logement.js

```diff
@@ -79,12 +74,13 @@ angular.module('ddsApp').controller('FoyerLogementCtrl', function($scope, $http,
         ['colocation', 'locationType', 'membreFamilleProprietaire', 'primoAccedant', 'loyer', 'charges', 'isChambre'].forEach(function(field) {
             delete logement[field];
         });
+        logement.loyer = 0;
     };
 
     $scope.updateCities = function updateCities() {
         $scope.retrievingCities = true;
 
-        $http.get('/api/outils/communes/' + $scope.postalCode)
+        $http.get('/api/outils/communes/' + $scope.logement.postalCode)
```

Comment:
If we stop using `postalCode` directly from `$scope`, please make sure it is not exposed in `$scope` anymore (it would be a good thing).